app/src/main/java/rec/games/pokemon/teambuilder/PokemonItemDetailActivity.java

```diff
@@ -153,7 +152,7 @@ public void shareToBrowser(){
 		}
 	}
 
-	public void ifItemAddedFAB(){
+	public void itemFAB(){
```

Comment:
Now it's even less clear what this method does. Please be descriptive, and use verbs. Something like toggleFABIcon, or addOrRemovePokemonFromTeam.

Reply:
Sorry about that. I have just updated using the second suggestion.

Reply:
Thanks. Much better :)